### ISSUE TITLE
Use base href from <base> tag

### ISSUE THREAD

**Issue by kravets-levko** [arkon/ng-inline-svg#48]
Hi @arkon!

Can the library use by-default base path from the `<base href="...">` tag? I think it makes sense, because using relative URLs, we expect that they'll be resolved in next order: 

1. relative to `<base>` tag;
2. relative to current URL path.

Currently the library has more powerful way to specify the base path, but this issue is about defaults.

I'm ready to implement this, but I decided to ask beforehand.

Thanks!

**Comment by arkon** [comment]
Hi @kravets-levko!

Sorry, this slipped through the cracks that is the black hole of my email inbox.

I think this would be a valuable option to have. If you want to submit a PR, I'd be happy to review it. :)

**Comment by kravets-levko** [comment]
Hi @arkon! I prepared a PR - arkon/ng-inline-svg/pull/49, and I kindly ask you to review it. Thanks!

**Comment by kravets-levko** [comment]
A short explanation how I encountered this issue: 
I have a directory structure like this:
```
dist/
  index.html
  assets/
     app.bundle.js
     logo.svg
     ...
```
And there fore `index.html` contains `<base href="/assets/">` - to allow loading resources from proper location. And when using this library like `<div [inlineSVG]="'logo.svg'"></div>` - it does not load file, because it creates absolute url without prefix from `<base>` tag - which is a bit confusing.

**Comment by arkon** [comment]
Published in v5.1.0.
